content/slack/index.md

```diff
@@ -0,0 +1,48 @@
+
+{{< button href="/schedule" >}}Join Slack{{< /button >}}
+</br>
+</br>
+
+## Channels
+
+Conversations on Slack are organized into channels that start with `#`. The following sections share details about the most common. Don't see one you want? Ask in `#mumblechatter`. You can even [create your own channel](https://slack.com/help/articles/201402297-Create-a-channel), such as for your Shield Lock or a one-time event!
+
+### Default channels
+
+Everyone automatically is part of the following channels:
+
+| Channel | Description | 
+| --- | --- |
+| #1stF | Anything fitness-related, including "preblasts" or promotional info for who's leading each workout the night before.|
+| #2ndF | Anything fellowship-related, such as happy hours, spontaneous meetups to watch a game, or recommendations on things to do.|
+| #3rdF | Anything faith-related, such as praises, prayers, and other ways we can help each other out.|
+| #backblasts | A channel where backblasts, or reports for each workout of the day, are automatically added based on the Q filling out the **Paxmate** bot. You can also review the [Paxmate Dashboard](/paxmate-dashboard/) based on these reports. |
+| #cot-announcements | Announcements that you would like the pax to hear about in Circle of Trust at the end of each workout. These are usually upcoming events like a race or service project. Only admins can post to this channel. If you have an announcement, let someone on the [Leadership](/leadership/) team know.|
+| #mumblechatter | A catch-all channel for general questions, comments, observations of life, etc. Usually the most active channel on the site.|
+
+### Other channels of interest
+
+Channels are often used to share interests or coordinate activities around a certain type of event. As Peak City takes off, we are trying to take a "narrow and deep" approach to encourage activity (vs. a "wide and shallow" large number of channels with fewer participants).
+
+> **Tip**: The F3 Carpex Slack (where Peak City started) has been around longer, so has a lot more people and channels. You might try posting there. For an invite, search "slack" on this [F3 Carpex page](https://f3carpex.com/links/).
+
+| Channel | Description | 
+| --- | --- |
+| #buy-sell-trade | Things you're selling or looking for, or deals that you found that other pax might be interested in, kind of like an F3 Marketplace. |
+| #diy-home-ec | Anything related to DIY projects, house projects, tools of the trade, recommendations for local service providers, and so on. |
+| #business-finance-career | Anything related to money, personal finances, career advice, jobs, professional development, and so on. |
+| #healthy-eats | Anything cooking, nutrition, diet-related. |
+
+## Moderation
+
+As with everything in F3, you are "Freed to Lead," and can post mostly whatever you like, or even help the [Commz team](/leadership/) moderate.
```

Comment:
Any other moderation guidelines we'd want?

Reply:
I think these are good. Only other I think of is call out explicitly spamming @channel and @here.